ci-operator/config/openshift-metal3/terraform-provider-ironic/openshift-metal3-terraform-provider-ironic-master.yaml

```diff
@@ -0,0 +1,52 @@
+base_images:
+  base:
+    name: "4.10"
+    namespace: ocp
+    tag: base
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  ocp_builder_rhel-8-golang-1.17-openshift-4.10:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.17-openshift-4.10
+binary_build_commands: make install
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+promotion:
+  name: test
```

Comment:
What's the reason for having a promote section here?

Reply:
No specific reason, probably it came with while I copied from some other repo :) and after you asking this maybe it is redundant?

Reply:
Yes, the `ocp-kni/test` it's currently just used to store the `dev-scripts` image (as it is not part of the ocp payload, but still required by some jobs), so better to remote this section